pyproject.toml

```diff
@@ -32,6 +32,10 @@ dependencies = [
     "docutils>=0.10,<0.20",
     "cryptography>=3.3.2,<38.0.5",
     "ruamel.yaml>=0.15.0,<=0.17.21",
+    # The clib wheel is generally downloaded as part of ruamel.yaml's
+    # dependency, but it's not being picked up so we explicitly add
```

Comment:
Or more specifically `ruamel.yaml` only requires `ruamel.yaml.clib` for less than or equal to Python 3.10. Adding `ruamel.yaml.clib` here ensures we have a consistent dependency closure across all Python versions. I think it would be include that so we are more specific on the rationale.